draft-poidt-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -397,40 +402,78 @@ This scenario, shown in {{figure-topo5}}and extends network topologies {{figure-
 
 # Operators' Use cases
 
+This section provides a set of packet over optical general use cases which are applicable to any network topologies in Section 4 and both for multi-layer networks using or not coherent pluggables in the routers. These use cases are presented following current operators’ priorities order.
 
-This section provides a set of packet over optical general use cases which are applicable to any network topologies in Section 4.
-These use cases are presented following current operators’ priorities order and several of them were already mentioned in IETF118 ccamp meeting back in November [draft-poidt-ccamp-actn-poi-pluggable-02].
-
-In general those use cases are the same:
--	 for a traditional packet over optical integration based on grey interfaces in the IP routers and use of transponders/muxponders in the optical domain
--	For a packet over optical integration considering coherent WDM pluggables in the IP routers over a media channel/Network Media channel in the optical domain
-The mention ‘valid for both’ has been added in the name of each use case else ‘valid for coherent pluggable’ when the use case is specific to the coherent pluggable approach.
+The use cases a generally applicable for both the traditional packet over optical integration based on grey interfaces in the IP routers and use of transponders/muxponders in the optical domain and for the packet over optical integration considering coherent WDM pluggables in the IP routers over a media channel/Network Media channel in the optical domain. For clarification purposes, the mention ‘valid for both’ has been added in the name of each use case else ‘valid for coherent pluggable’ when the use case is specific to the coherent pluggable approach.
 
 ## 1.	End-to-end multi-layer visibility and management (valid for both)
 
 ### a.	End-to-end multi-layer network and service topology discovery
-Target is to have a full end-to-end multi-layer view from all the layers and their dependencies: service layer (e.g. L3VPN/L2VPN), transport layer (RSVP-TE, SR-TE), IP layer (IGP), Ethernet layer, OTN L1 layer (optional), DWDM L0 layer, OMS, OTS and fibre.
-The above discovery includes inventory discovery by each controller and exposure to the MDSC of the required information for a complete end-to-end multi-layer view of the network
+Target is to have a full end-to-end multi-layer view from all the layers and their inter-dependencies: service layer (e.g. L3VPN/L2VPN), transport layer (RSVP-TE, SR-TE), IP layer (IGP), Ethernet layer, OTN L1 layer (optional), photonic L0 layer (OCh, OMS, OTS and fibre).
+The above discovery includes inventory discovery by each controller and exposure to the MDSC of the required information for a complete end-to-end multi-layer view of the network.
+
+#### Coherent DWDM pluggable insertion in the router linecard port
```

Comment:
add "('valid for coherent pluggable')" at the end of the title

Reply:
done!